src/Native/System.Native/pal_networking.cpp

```diff
@@ -2329,8 +2343,13 @@ static Error CloseSocketEventPortInner(int32_t port)
 static Error TryChangeSocketEventRegistrationInner(
     int32_t port, int32_t socket, SocketEvents currentEvents, SocketEvents newEvents, uintptr_t data)
 {
+#ifdef EV_RECEIPT
     const uint16_t AddFlags = EV_ADD | EV_CLEAR | EV_RECEIPT;
     const uint16_t RemoveFlags = EV_DELETE | EV_RECEIPT;
+#else
+    const uint16_t AddFlags = EV_ADD | EV_CLEAR;
+    const uint16_t RemoveFlags = EV_DELETE;
```

Comment:
I think this may have been asked before, but what are the ramifications of not having EV_RECEIPT here?